man/man8/zpool-events.8

```diff
@@ -97,6 +97,17 @@ This can be an indicator of problems with the underlying storage device.
 The number of delay events is ratelimited by the
 .Sy zfs_slow_io_events_per_second
 module parameter.
+.It Sy dio_verify
+Issued when there was a checksum verify error after a Direct I/O write has been
+issued but not committed to disk.
+This event can only take place if the module parameter
+.Sy zfs_vdev_direct_write_verify_cnt
```

Comment:
We may want to add `zfs_vdev_direct_write_verify_cnt` itself in man pages too, sorry if I missed it

Reply:
Yep, I've missed it, great